<launch>
    <arg name="config_pkg" default="$(eval find('kingfisher_description'))"/>

    <group ns="kf">
        <node
                pkg="tauv_common"
                type="dynamics_parameter_estimator"
                name="dynamics_parameter_estimator"
                output="screen"
        >
            <rosparam command="load" file="$(arg config_pkg)/yaml/dynamics_parameter_estimator.yaml" />
        </node>
    </group>
</launch>
